CONTRIBUTING.md

```diff
@@ -1,8 +1,56 @@
 # Contributing
 
-Thank you for wanting to contribute back to the Sensu Community repository. Here are a few guidelines to follow as you do so: 
+Thank you for wanting to contribute back to the Sensu Community. Here are a few guidelines to follow to open an issue here:
 
 1. All contributors are expected to read and follow [our code of conduct](https://sensuapp.org/conduct). Anyone can reach out to administrators at community@sensu.io if needed.
 2. This repository is designed to be an aggregator of resources for our community. Feel free to extend that idea through new documents or suggestions. We welcome your creativity.
 
+# Contributing to the Plugins & Extensions organizations
+
+We need participation like yours in order to maintain the massive number of plugins and extensions that make Sensu an amazing project. Our ambitious goal is to:
+
+* Have at least 2 organizational-wide maintainers at all times
+* Have at least 1 subject matter expert (SME) with write access to all plugins
```

Comment:
an SME on all plugins? or each repo should have an SME? I know this is a subtle difference in words but they mean very different things.

Reply:
Great clarification. I mean on each individual plugin and will fix the wording.